server/index.js

```diff
@@ -83,4 +83,35 @@ app.get('/itemsIncluded/:tierId', (req, res) => {
   })
 });
 
+app.get('/itemsByBundleId/:bundleId', (req, res) => {
+  const dataString = req.params;
+  return db.tiersRequest(dataString)
+  .then(tiersData => {
+    if (tiersData) {
+      return tiersData;
+    } else {
+      res.status(404).send('no such bundleId');
```

Comment:
Consider whether handling this question of valid bundle ID first before making a db call makes sense. That way you can just throw an error to begin with and shoot directly to a `catch` block (which would need to be added) & avoid unnecessary db calls.

I think that would eliminate the need for this `.then` block entirely, since the alternate path is just passing the argument through.